### DISCUSSION TITLE
Transparent Encryption Similar to Git-Crypt Possible with JJ?

### DISCUSSION THREAD

**Discussion by andar1an** [jj-vcs/jj#4995] [Q&A] [answered]
Is it possible to use git-crypt or similar with JJ, or are there plans to enable transparent encryption with JJ?

**Comment by arxanas** [comment, marked as answer]
It's currently not possible and I believe nobody has proposed a design or implementation yet. I think we would like to support it, at least for Git compatibility.

The same underlying mechanism, once implemented, might also be used to handle LFS, submodules, end-of-line differences, merge drivers, etc.
